tasks/autodetect.yml

```diff
@@ -26,7 +26,7 @@
           {%- else -%}
           qemu
           {%- endif -%}
-  when: libvirt_vm_engine is none
+  when: libvirt_vm_engine is none or libvirt_vm_engine | length == 0
```

Comment:
could we not use `when: libvirt_vm_engine`?

Reply:
I think there are cases when ansible doesn't like that. We need an [experiment](https://github.com/markgoddard/ansible-experiments) for that...